yarn.lock

```diff
@@ -1485,45 +1485,12 @@
     "@types/node" ">= 8"
 
 "@percolate/eslint-plugin@link:./pkgs/eslint-plugin":
-  version "1.1.12"
-  dependencies:
-    "@typescript-eslint/eslint-plugin" "2.7.0"
-    "@typescript-eslint/parser" "2.7.0"
-    eslint-import-resolver-node "0.3.2"
-    eslint-plugin-eslint-comments "3.1.2"
-    eslint-plugin-filenames "1.3.2"
-    eslint-plugin-import "2.18.2"
-    eslint-plugin-lodash "6.0.0"
-    eslint-plugin-react "7.16.0"
-    eslint-plugin-react-hooks "2.2.0"
-    eslint-plugin-simple-import-sort "7.0.0"
-    html-tags "3.1.0"
+  version "0.0.0"
+  uid ""
 
 "@percolate/kona@link:./pkgs/kona":
-  version "3.4.4"
-  dependencies:
-    "@commitlint/config-conventional" "12.1.1"
-    "@commitlint/format" "12.1.1"
-    "@commitlint/lint" "12.1.1"
-    "@commitlint/load" "12.1.1"
-    "@percolate/cli-utils" "0.1.10"
-    commitizen "4.2.3"
-    fast-glob "3.2.5"
-    find-up "4.1.0"
-    husky "3.0.9"
-    jest "26.6.3"
-    jest-junit "10.0.0"
-    jest-watch-typeahead "0.5.0"
-    lcov-parse "1.0.0"
-    lodash.clonedeep "4.5.0"
-    lodash.isequal "4.5.0"
-    lodash.template "4.5.0"
-    lodash.uniq "4.5.0"
-    micromatch "4.0.2"
-    p-map "4.0.0"
-    parse-diff "0.7.1"
-    yargs "16.2.0"
-    yarn-deduplicate "1.1.1"
+  version "0.0.0"
```

Comment:
Why the diff?

Reply:
Dependabot

Reply:
dependencies intro'd by dependabot [update](https://github.com/percolate/blend/commit/47f61e1968fb2c031de47896b47c7306495020df). Not clear how. 

Original PR adding `"@percolate/kona@link:./pkgs/kona":` is here: https://github.com/percolate/blend/pull/38/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR1797

Marking as resolved.